features/steps/cli_steps.py

```diff
@@ -565,14 +576,7 @@ def check_jupyter_nb_proc_on_port(context: behave.runner.Context, port: int):
     """
     url = f"http://localhost:{port}"
     try:
-        util.wait_for(
-            func=_check_service_up,
-            context=context,
-            url=url,
-            string="Jupyter Notebook",
-            timeout_=15,
```

Comment:
I'm guessing we don't need this timeout here anymore because the test now waits for the notebook to be up before checking the port?

Reply:
Exactly, we don't need to be constantly flooding the server with requests, rather just wait for the server to tell us when it is ready.